build.sbt

```diff
@@ -68,14 +76,18 @@ lazy val anorm = project
       // New functions
       missMeth("anorm.Sql.unsafeStatement$default$2"),
       missMeth("anorm.Sql.unsafeResultSet"),
-      missMeth("anorm.Sql.unsafeStatement")),
+      missMeth("anorm.Sql.unsafeStatement"),
+      missMeth("anorm.ToStatement.contramap"),
```

Comment:
Not sure I understand your point. If you recompile the library of course it will work. I'm not saying anything won't compile. The problem, like I said, is if you use a library like anorm-relational that's compiled with an older anorm version, its implementations of the `ToStatement` trait would be incomplete. This would likely cause an error at runtime.